framework/test/mocha/integration/blocks/process/on_receive_block.js

```diff
@@ -187,12 +187,15 @@ describe('integration test (blocks) - process receiveBlockFromNetwork()', () =>
 	}
 
 	function getKeypair(passphrase) {
-		return library.ed.makeKeypair(
-			crypto
-				.createHash('sha256')
-				.update(passphrase, 'utf8')
-				.digest()
-		);
+		const {
+			publicKeyBytes: publicKey,
```

Comment:
Just wondering why we are renaming the `publicKeyBytes` to `publicKey`.
Maybe use the naming with bytes as that better represents the content of the variable?

Reply:
I've renamed it to keep the same interface as before.